src/vma/dev/ring_simple.cpp

```diff
@@ -83,7 +83,15 @@ inline void ring_simple::send_status_handler(int ret, vma_ibv_send_wr* p_send_wq
 
 qp_mgr* ring_eth::create_qp_mgr(const ib_ctx_handler* ib_ctx, uint8_t port_num, struct ibv_comp_channel* p_rx_comp_event_channel) throw (vma_error)
 {
+#if !defined(DEFINED_VMAPOLL) && defined(HAVE_INFINIBAND_MLX5_HW_H)
+	if (strstr(((ib_ctx_handler*)ib_ctx)->get_ibv_device()->name, "mlx5")) {
```

Comment:
Please substitute this for a macro or a separate function that tests if the NIC is mlx5.